src/attendee/attendee.service.ts

```diff
@@ -0,0 +1,37 @@
+import { Injectable } from '@nestjs/common';
+import { PrismaClient } from '@prisma/client'
+import { AttendeeWithUser } from '../../prisma/types';
+
+const prisma = new PrismaClient()
+
+@Injectable()
```

Comment:
nestjs is a modular framework, I would also create a attendes.module.ts to correctly inject this. 
I think you are importing this into the events module, but what you should import is the attendesModule (with the exported attendee.service.ts)